tests/ui/threads-sendsync/tls-dont-move-after-init.rs

```diff
@@ -0,0 +1,38 @@
+//@ run-pass
+#![allow(stable_features)]
```

Comment:
What's this for?

Reply:
Nothing, it was left over after copying `tls-init-on-init.rs` as a template.